packages/manager/modules/pci/src/projects/onboarding/translations/Messages_fr_FR.json

```diff
@@ -14,5 +14,8 @@
   "pci_projects_onboarding_us_banner_trusted_customer_info_1": "Vous souhaitez plus d’informations ou passer commande ?",
   "pci_projects_onboarding_us_banner_trusted_customer_info_2": "Demandez qu’un conseiller OVHcloud vous rappelle gratuitement",
   "pci_projects_onboarding_banner_kyc_validation_under_verification_description": "Vous souhaitez créer un projet Public Cloud : la réglementation en vigueur nécessite une vérification de votre identité. Le traitement de vos documents est actuellement en cours d'analyse par notre équipe. Pour plus d'information, vous pouvez consulter ",
-  "pci_projects_onboarding_banner_kyc_validation_under_verification_link": "Mes demandes d'assistance"
+  "pci_projects_onboarding_banner_kyc_validation_under_verification_link": "Mes demandes d'assistance",
+  "pci_projects_onboarding_italy_agreements_label": "Il Cliente dichiara di riconoscere, accettare ed approvare espressamente, ai sensi dell'art. 1341, secondo comma, e dell'art. 1342, secondo comma, del Codice Civile",
+  "pci_projects_onboarding_italy_agreements_collapsible_title": "Le specifiche clausole indicate qui",
+  "pci_projects_onboarding_italy_agreements_collapsible_description": "Il Cliente dichiara di riconoscere, accettare ed approvare espressamente, ai sensi dell'art. 1341, secondo comma, e dell'art. 1342, secondo comma, del Codice Civile, le seguenti specifiche clausole del presente Contratto: 2.5 (“Chiusura dell’Account Cliente”), 3.7 (“Sospensione dei Servizi”), 4.2 (“SLA”), 4.3 (“Crediti di servizio”), 5.2 (“Modifica dei Servizi ordinati”), 7.1 (“Casi di e-mail fraudolente e indesiderate”), 7.3 (“Sospensione e disattivazione”), 10.2 (“Responsabilità di OVHcloud”), 10.3 (“Limitazioni di responsabilità”), 10.4 (“Eccezioni”), 10.6 (“Responsabilità del Cliente”), 10.7 (“Garanzie”), 10.8 (“Forza Maggiore”), 11.2 (“Modifica delle tariffe”), 11.6 (“Mancato pagamento e ritardo nel pagamento”), 11.7 (“Contestazione”), 12.2 (“Rinnovo dei Servizi”), 14.1 (“Legge applicabile e foro competente”), 14.2, ult. cpv. (“Sanzioni internazionali e Controllo delle esportazioni”), 14.7 (“Modifiche al Contratto”), 14.9 (“Cessione del contratto”)."
```

Comment:
If these two texts are not translated, shouldn't we use a constant ?

Reply:
I there is a need to do so. This exception is handle on the translation team side.

Reply:
I don't understand, from the ticket it seems that this will only appear for IT subs, but subs =/= language
So if it stays like that, an IT user with the french language will have the page in french and this text in italian

Reply:
@JacquesLarique  I've put the text as a constant.